### DISCUSSION TITLE
Add class(es) to Asset::getImg()

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [craftcms/cms#11561] [Q&A] [answered]
Is there any way I can add classes to the `getImg()` function?

I can't seem to find this in the documentation.

**Comment by brandonkelly** [comment, marked as answer]
You can add classes using the [`|attr`](https://craftcms.com/docs/4.x/dev/filters.html#attr) filter:

```twig
{{ asset.img|attr({ class: 'foo bar' })
```
